README.md

```diff
@@ -56,6 +56,43 @@ The `projector_bin` path is added to the `PATH` variable when in the project or
 it's child directories. The `projector_bin` path is removed from `PATH` when
 leaving the project directory.
 
+# Configure
```

Comment:
This is duplicated with `doc/section/configure.md`. What's that all about?

Reply:
Reading back the documentation I must conclude that I was very tired when writing this. Still needs some work